pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 requires-python = ">= 3.9"
 dependencies = [
     "numpy",
-    "cupy",
+    "cupy!=13.0.*", #see https://github.com/SBC-Utrecht/pytom-match-pick/issues/106
```

Comment:
```suggestion
    "cupy!=13.0.*, #see https://github.com/SBC-Utrecht/pytom-match-pick/issues/106
```
Maybe the double quotes also need to be removed?

Reply:
Nope, the test failed because of lingering processes, we probably have to up the sleep on our parallel test

Reply:
I saw that sometime before as well. Some of these leaked semophore object thingies